src/stage.h

```diff
@@ -0,0 +1,39 @@
+#ifndef _STAGE_H
+#define _STAGE_H
+#define WIN32_LEAN_AND_MEAN
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include "constants.h"
+
```

Comment:
But i only have coordinates. Which is just the indexes. Having 2 will only screw us over